m2m-transfers/REPWRITERSPECS/BANNO.M2MTRANSFERS.V1.POW

```diff
@@ -2039,6 +2057,8 @@ PROCEDURE INITIALIZEDATA
  LFNUMBER=0
  ERRORCODE=0
  PREFLOC=@PREFLOCATOR
+ PARAMTESTMODE=FALSE
```

Comment:
These 2 variables are being reset in READCONFIGFILESETTINGS. Makes more sense there as that is where the rest of the PARAM* variables are being reset. I would suggest removing them from here.

Reply:
Removed